Sources/ScrollViewProxy/ScrollViewProxy.swift

```diff
@@ -71,10 +71,10 @@ public struct ScrollViewReader<ID: Hashable, Content: View>: View {
                     self.proxy.save(geometry: preference.geometry, for: preference.id)
                 }
             }
-            .onPreferenceChange(ScrollViewProxyPreferenceKey<ID>.self) { _ in
-                // seems this will not be called due to ScrollView/Preference issues
-                // https://stackoverflow.com/a/61765994/3019595
-            }
+//            .onPreferenceChange(ScrollViewProxyPreferenceKey<ID>.self) { _ in
+//                // seems this will not be called due to ScrollView/Preference issues
+//                // https://stackoverflow.com/a/61765994/3019595
+//            }
```

Comment:
@Amzd 
So, I tried using it without this line. It seems that if we remove it, then `transformPreference` is not called. I'm not sure why exactly.

Reply:
Ah that's weird, I thought it didn't do anything.

Reply:
I also thought it didn't do anything.